arrows/vxl/average_frames.cxx

```diff
@@ -0,0 +1,722 @@
+/*ckwg +5
+ * Copyright 2012-2019 by Kitware, Inc. All Rights Reserved. Please refer to
+ * KITWARE_LICENSE.TXT for licensing information, or contact General Counsel,
+ * Kitware, Inc., 28 Corporate Drive, Clifton Park, NY 12065.
+ */
+
+#include "average_frames.h"
+
+#include <vital/util/ring_buffer.h>
+#include <arrows/vxl/image_container.h>
+
+#include <vil/vil_pixel_format.h>
+#include <vil/vil_image_view.h>
+#include <vil/vil_convert.h>
+#include <vil/vil_math.h>
+
+#include <exception>
+#include <memory>
+
+namespace kwiver {
+namespace arrows {
+namespace vxl {
+
+namespace {
+
+/// Base class for all online frame averagers instances
```

Comment:
This is one of those cases that doesn't need anything special from VXL.  It's just simple math operations on images which KWIVER can easily do.  No need to revise on this branch.  But at some point I'd like to see a new "image processing" arrow that has operations like this without requiring VXL.